File: PYTHON PROGRAMMING/Test30.py
# 30 PRINT WHETHER THE NUMBER IS DIVISIBLE BY THE GIVEN TARGET OR NOT
def func(ip, target):
    if ip % target == 0:
        print(f"{ip} is divisible by {target}")
    else:
        print(f"{ip} is not divisible by {target}")
func(36, 3)
func(18, 4)

File: TYPESCRIPT PROGRAMMING/Test30.ts
// 30. PRINT WHETHER THE NUMBER IS DIVISIBLE BY THE GIVEN TARGET OR NOT
const func30 = (ip: number, target: number): void => {
    let res: string = (ip % target) ? `${ip} Is Divisible By ${target}` : `${ip} Is Not Divisible By ${target}`;
    console.log(res);
}
func30(27, 3);
func30(27, 2);